### PR TITLE
Gibbon response fixes

### PR DESCRIPTION
**Pull request by SergeyMell**

### CODE REVIEW

**Comment by vinay-mittal** [comment]
@SergeyMell I am verifying this pull request. The changes you have done in `Gibbon` API requests are not given on their README. I will hit these requests and verify the responses as given by you and proceed ahead.

File: app/services/gibbon_service/list_service.rb [hunk 1]
```diff
@@ -55,17 +59,18 @@ def unsubscribe_members(unsubscribable_uids = [])
...
 
     private
-      def retrieve_members
-        p "retrieving members for list #{@list_uid}"
-        @members = gibbon.lists(@list_uid).members.retrieve(DEFAULT_MEMBER_RETRIEVAL_PARAMS)['members']
-      end
+
+    def retrieve_members
+      p "retrieving members for list #{@list_uid}"
+      @members = gibbon.lists(@list_uid).members.retrieve(DEFAULT_MEMBER_RETRIEVAL_PARAMS)['members']
```
Comment (vinay-mittal) [inline app/services/gibbon_service/list_service.rb]: @SergeyMell Here `.body` has not been used. Is there any specific reason for it?

Comment (SergeyMell) [reply app/services/gibbon_service/list_service.rb]: It's your original code. Just moved a couple of spaces to the left. But `body` should be there either. I'll add this.

**Comment by SergeyMell** [comment]
Thank you @vinay-mittal. I didn't check the readme but I've just tested the requests and run into this issue. Please, let me know if any additional data required.

**Comment by SergeyMell** [comment]
@vinay-mittal, I've fixed the one missed `body`. Please, take a look.
By the way, did you have a chance to test my changes?

**Comment by vinay-mittal** [comment]
@SergeyMell  I verified this change and seems right. They changed it in Gibbon version 3+. It is specified [here](https://github.com/amro/gibbon#important-notes) . And in `master` branch gibbon version used is `3.2.0`, so it spree versions `less than 3`. Gibbon version is `2.3` so it will not work there.

Thanks for the contribution. :)

**Pull request merged**